qa/L0_doa/test.sh

```diff
@@ -17,18 +17,41 @@ source ../common/util.sh
 
 rm -f test.log
 
+# Set test parameters
 MODEL_ANALYZER=`which model-analyzer`
 
-RET=0
+python3 test_config_generator.py
+
+LIST_OF_CONFIG_FILES=(`ls | grep .yml`)
 
-set +e
-run_analyzer
-if [ $? != 1 ]; then
-    echo -e "\n***\n*** Failed to run model-analyzer. \n***"
-    cat $ANALYZER_LOG
+if [ ${#LIST_OF_CONFIG_FILES[@]} -le 0 ]; then
+    echo -e "\n***\n*** Test Failed. No config file exists. \n***"
     RET=1
+    exit $RET
 fi
-set -e
+
+# Run the analyzer and check the results
+RET=0
+
+for CONFIG_FILE in ${LIST_OF_CONFIG_FILES[@]}; do
```

Comment:
This test really doesn't need a list of config files. Please revert this to the previous state.

Reply:
Agree